pytorch_lightning/accelerators/hpu.py

```diff
@@ -0,0 +1,55 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any, Dict, List, Union
+
+import torch
+
+from pytorch_lightning.accelerators.accelerator import Accelerator
+from pytorch_lightning.utilities import _HPU_AVAILABLE
+from pytorch_lightning.utilities.rank_zero import rank_zero_debug
+
+
+class HPUAccelerator(Accelerator):
+    """Accelerator for HPU devices."""
+
+    @staticmethod
+    def name() -> str:
```

Comment:
@kaushikb11 isn't this getting removed with the registry?

Reply:
Done!